isobus/include/isobus/isobus/isobus_virtual_terminal_client.hpp

```diff
@@ -1606,9 +1594,6 @@ namespace isobus
 
 		// Status message contents from the VT
 		std::uint32_t lastVTStatusTimestamp_ms = 0; ///< The timestamp of the last VT status message
-		std::uint16_t activeWorkingSetDataMaskObjectID = NULL_OBJECT_ID; ///< The active working set data mask object ID
-		std::uint16_t activeWorkingSetSoftKeyMaskObjectID = NULL_OBJECT_ID; ///< The active working set's softkey mask object ID
-		std::uint8_t activeWorkingSetMasterAddress = NULL_CAN_ADDRESS; ///< The active working set master address
```

Comment:
These still seem useful to me in the VTClient, since I think using the helper should be optional